pkg/types/machinepools.go

```diff
@@ -39,6 +47,9 @@ type MachinePool struct {
 	// +optional
 	// Default is for hyperthreading to be enabled.
 	Hyperthreading HyperthreadingMode `json:"hyperthreading,omitempty"`
+
+	// Architecture is the instruction set architecture of the machine pool.
+	Architecture Architecture `json:"architecture"`
```

Comment:
`omitempty`?

Reply:
Say "Defaults to amd64." for backwards-compat with existing configs?  Or "If unset, the installer may choose a default value, and the default value may evolve over time."?  Or some such?